backend/README.md

```diff
@@ -4,6 +4,8 @@ Refer to [top level README](https://github.com/City-of-Helsinki/yjdh/blob/main/R
 
 [kesaseteli/README.md](https://github.com/City-of-Helsinki/yjdh/blob/main/backend/kesaseteli/README.md) or [benefit/README.md](https://github.com/City-of-Helsinki/yjdh/blob/main/backend/benefit/README.md) contain more information about the local setup. This applies to TET pretty well.
 
+Django updated to version 4.2 during april 2024, as security support for 3.2 ends 30.4.2024. Django 4.2 will stop receiving security updates 30.4.2026.
```

Comment:
This is in `backend/README.md` and tomorrow is the last day of April, so this sounds overly optimistic for all backends.

Reply:
Yeah, I this PR was stuck in the pipelines for a couple of weeks. I will update the readme after we have a realistic schedule for the update.